components/help-dropdown/help-dropdown.scss

```diff
@@ -0,0 +1,11 @@
+.custom-select {
```

Comment:
This is for custom select elements across the site, not just for the `help-dropdown` component. So let's put this somewhere/name it more appropriately?